static/sass/_pattern_footer.scss

```diff
@@ -1,268 +1,302 @@
 // Footer pattern styling
 
 @mixin footer-title {
-  color: $color-mid-dark;
-  cursor: pointer;
-  line-height: 1.5;
-  margin-bottom: 0;
+	color: $color-mid-dark;
```

Comment:
This file has had spaces converted to tabs. Check your editor settings and revert to spaces, please.